everware/spawner.py

```diff
@@ -81,39 +82,6 @@ def lister(mm):
         else:
             return m(*args, **kwargs)
 
-    _git_executor = None
-    @property
-    def git_executor(self):
-        """single global git executor"""
-        cls = self.__class__
-        if cls._git_executor is None:
-            cls._git_executor = ThreadPoolExecutor(1)
-        return cls._git_executor
-
-    _git_client = None
-    @property
-    def git_client(self):
-        """single global git client instance"""
-        cls = self.__class__
-        if cls._git_client is None:
-            cls._git_client = git.Git()
-        return cls._git_client
-
-    def _git(self, method, *args, **kwargs):
-        """wrapper for calling git methods
-
-        to be passed to ThreadPoolExecutor
-        """
-        m = getattr(self.git_client, method)
-        return m(*args, **kwargs)
-
-    def git(self, method, *args, **kwargs):
```

Comment:
I think it would be good to keep this as a shortcut to `self._git_executor.git()`